src/Components/Main/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import data from "../../data";
+class Main extends Component {
+  state = {
+    score: 0,
+    timer: 60,
+    counter: 0,
+    currentImg: data[0].img,
+    currentName: data[0].name,
+    answer: ""
+  };
+
+  compare = event => {
+    const { value } = event.target;
+    const { currentName, counter, score } = this.state;
+    this.setState({ answer: value }, () => {
+      if (currentName === this.state.answer) {
+        this.setState(prevState => ({
+          score: prevState.score + 1,
+          currentName: data[prevState.counter + 1].name,
+          currentImg: data[prevState.counter + 1].img,
+          answer: "",
+          counter: counter + 1
+        }));
+      }
+    });
+  };
+
+  render() {
+    return (
```

Comment:
You can destructure all values from this.state e.g.
`const {timer, score, answer} = this.state